src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,111 @@
+            if (resultSet.next()) {
+                Long idOfManufacturer = resultSet.getLong("id");
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                manufacturer = new Manufacturer(idOfManufacturer, name, country);
+            }
+
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public List<Manufacturer> getAll() throws RuntimeException {
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 Statement getAllStatement = connection.createStatement()) {
+            ResultSet resultSet = getAllStatement
+                    .executeQuery(
+                        "SELECT manufacturers.id, manufacturers.name, manufacturers.country"
+                        + "  FROM manufacturers WHERE is_deleted = 0");
+            while (resultSet.next()) {
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                Long id = resultSet.getObject("id", Long.class);
+                Manufacturer manufacturer = new Manufacturer(id, name, country);
+                manufacturerList.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
+        }
+        return manufacturerList;
+    }
+
+    @Override
+    public void delete(Long id) {
+        String deleteRequest = "UPDATE manufacturers SET is_deleted = 1 where id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement deleteStatement =
+                         connection.prepareStatement(
+                                 deleteRequest, Statement.RETURN_GENERATED_KEYS)) {
+            deleteStatement.setLong(1, id);
+            deleteStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
+        }
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateRequest = "UPDATE manufacturers SET name = ?, country = ? WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement updateStatement =
+                        connection.prepareStatement(
+                                updateRequest, Statement.RETURN_GENERATED_KEYS)) {
+            updateStatement.setString(1, manufacturer.getName());
+            updateStatement.setString(2, manufacturer.getCountry());
+            updateStatement.setLong(3, manufacturer.getId());
+            updateStatement.executeUpdate();
+            return new Manufacturer();
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
+        }
+    }
+}
```

Comment:
It is better to use special private method for similar operation in code and follow DRY principle.
```suggestion
    private Manufacturer parseResultSet(ResultSet resultSet) throws SQLException {
        return new Manufacturer(resultSet.getObject("id", Long.class),
                                resultSet.getString("name"),
                                resultSet.getString("country"));
    }
}
```